src/bin/pgcopydb/cli_compare.c

```diff
@@ -301,11 +312,23 @@ cli_compare_schema(int argc, char **argv)
 		exit(EXIT_CODE_INTERNAL_ERROR);
 	}
 
+	if (!IS_EMPTY_STRING_BUFFER(compareOptions.snapshot) && !copydb_set_snapshot(&copySpecs))
```

Comment:
nitpicking: please make it fit in two lines, or even with embedded if statements, for ease of maintenance and also to respect line length (does it fit within 80 chars, per project policy).